troi/acousticbrainz/annoy.py

```diff
@@ -52,7 +52,7 @@ def read(self, inputs):
             raise PipelineError("Cannot fetch annoy similarities from AcousticBrainz: Invalid JSON returned: " + str(err))
 
         entities = []
-        for row in results:
+        for row in results[self.mbid]["0"]:
```

Comment:
I don't think the Annoy element supports specifying custom offsets for mbids yet so we can safely hardcode the "0".

Reply:
If you don't specify an offset in the query then it defaults to 0. Hard-coding here is fine